examples/dataservice-read/example.cpp

```diff
@@ -32,29 +32,101 @@
 #include <olp/dataservice/read/PartitionsRequest.h>
 
 namespace {
-const std::string gKeyId("");      // your here.access.key.id
-const std::string gKeySecret("");  // your here.access.key.secret
-const std::string gCatalogHRN(
-    "hrn:here:data:::here-optimized-map-for-visualization-2");
-const std::string gLayerId("omv-base-v2");
-const std::string gPartitionId("100000093");  // Helsinki, Finland
+const std::string kKeyId("");      // your here.access.key.id
+const std::string kKeySecret("");  // your here.access.key.secret
+const std::string kCatalogHRN("hrn:here:data:::edge-example-catalog");
+constexpr size_t kMaxLayers(5);
+constexpr size_t kMaxPartitions(5);
+constexpr auto kLogTag = "read-example";
+
+// handles catalog response and returns a partition ID from it to be processed
+// in later requests
+std::string handleCatalogResponse(
```

Comment:
Method names should be CamelCase not camelCase
See https://google.github.io/styleguide/cppguide.html#Function_Names
Apply please for all below.